update.sh

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env bash
 
+SVDTOOLS="${SVDTOOLS:-svd}"
+
 set -ex
 
-cargo install --version 0.21.0 svd2rust
-cargo install --version 0.8.0  form
+# Path to `svd`/`svdtools`
+
+cargo install --version 0.25.1 svd2rust
```

Comment:
```suggestion
cargo install --version 0.27.0 svd2rust
```
This fixes the issue mentioned in https://github.com/rp-rs/rp2040-pac/pull/61#issuecomment-1646673324

Reply:
This also requires `critical-section`. So this should be added to Cargo.toml:
```
[dependencies.critical-section]
optional = true
version = "1.0.0"
```

Reply:
Just commit in my branch, please.